lib/headquarters/api/endpoints.rb

```diff
@@ -0,0 +1,12 @@
+module Headquarters
+  class API
+    module Endpoints
```

Comment:
The whole point is that by looking at the API class I don't know all it has, and that may lead to confusion. If I could see

```ruby
class API
  include Endpoints
  include Factory
  ...
end
```

I would at least know that there are methods being mixed in.